### PR TITLE
add web app

### PR DESCRIPTION
**Pull request by jburel**
Add the apps to the job. This PR adds the changes made to merge-ci.
Note that it  includes omero-web-zarr

cc @will-moore

### CODE REVIEW

File: home/jobs/OMERO-web/config.xml [hunk 1]
```diff
@@ -181,6 +179,32 @@ pip install omero-webtagging-tagsearch
...
+
+# virtual microscope
+pip install git+https://github.com/$SPACE_USER/omero-virtual-microscope.git@$MERGE_PUSH_BRANCH#egg=omero-virtual-microscope
+omero config append omero.web.apps &apos;&quot;virtualmicroscope&quot;&apos;
+omero config set omero.web.login_redirect &apos;{&quot;redirect&quot;: [&quot;webindex&quot;], &quot;viewname&quot;: &quot;webindex_custom&quot;}&apos;
+
+
+# CORS
+pip install &quot;django-cors-headers==3.2.1&quot;  # 3.7.0 for Django 3.2
+omero config append omero.web.apps &apos;&quot;corsheaders&quot;&apos;
```
Comment (jburel) [inline home/jobs/OMERO-web/config.xml:202]: minimally the configuration needs to be added to the slc python38 branch

File: home/jobs/OMERO-web/config.xml [hunk 2]
```diff
@@ -181,6 +179,32 @@ pip install omero-webtagging-tagsearch
...
+omero config set omero.web.signup.email.enabled True
+
+# virtual microscope
+pip install git+https://github.com/$SPACE_USER/omero-virtual-microscope.git@$MERGE_PUSH_BRANCH#egg=omero-virtual-microscope
+omero config append omero.web.apps &apos;&quot;virtualmicroscope&quot;&apos;
+omero config set omero.web.login_redirect &apos;{&quot;redirect&quot;: [&quot;webindex&quot;], &quot;viewname&quot;: &quot;webindex_custom&quot;}&apos;
+
+
+# CORS
+pip install &quot;django-cors-headers==3.2.1&quot;  # 3.7.0 for Django 3.2
```
Comment (sbesson) [inline home/jobs/OMERO-web/config.xml:201]: That's effectively a downgrade from https://github.com/ome/omero-web/blob/59ec465df1a4f55c7097696647e14dccc810a602/setup.py#L59. Is this still a requirement?

**Review by sbesson — COMMENTED** [review]
One dependency question. Otherwise, looks fine to me.

File: home/jobs/OMERO-web/config.xml (hunk 2 above)
Comment (jburel) [reply home/jobs/OMERO-web/config.xml:201]: The PR was opened a while back. So that needs to be adjusted

Comment (jburel) [reply home/jobs/OMERO-web/config.xml]: see [fdb265e](https://github.com/ome/devspace/pull/192/commits/fdb265ec85f2cc24f97cac8e2e19af28fcac2e95)

**Review by sbesson — APPROVED** [review]

**Pull request merged**